### PR TITLE
DOCS-1466: Add consistent vis names

### PR DESCRIPTION
**Pull request by andf-viam**
Standardize vision service names on `<vision-service-name>` for templates and `my_vision_service` for examples.
- Leaving screenshot (step 4) alone for now, we will return to App UI screenshots after upcoming robot details page redesign, I think.

### CODE REVIEW

**Comment by andf-viam** [comment]
@stevebriskin Hi Steve super simple vision service name standardization for these examples, per your ticket. Let me know if you'd like to see any changes -- thanks!

File: docs/ml/vision/detection.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ In your vision service's panel, fill in the **Attributes** field.
 
 ```json {class="line-numbers linkable-line-numbers"}
 {
-  "mlmodel_name": "<detector_name>"
+  "mlmodel_name": "<vision-service-name>"
```
Comment (stevebriskin) [inline docs/ml/vision/detection.md:184]: i think this is incorrect...should the ml service name i believe

File: docs/ml/vision/detection.md [hunk 2]
```diff
@@ -197,7 +197,7 @@ Add the vision service object to the services array in your raw JSON configurati
     "type": "vision",
     "model": "mlmodel",
     "attributes": {
-      "mlmodel_name": "<detector_name>"
+      "mlmodel_name": "<vision-service-name>"
```
Comment (stevebriskin) [inline docs/ml/vision/detection.md:200]: same

File: docs/ml/vision/detection.md [hunk 3]
```diff
@@ -213,7 +213,7 @@ Add the vision service object to the services array in your raw JSON configurati
     "type": "vision",
     "model": "mlmodel",
     "attributes": {
-      "mlmodel_name": "person_detector"
+      "mlmodel_name": "my_vision_service"
```
Comment (stevebriskin) [inline docs/ml/vision/detection.md:216]: same. but why no <>?

File: docs/ml/vision/detection.md [hunk 4]
```diff
@@ -247,7 +247,7 @@ If you intend to use the detector with a camera that is part of your robot, you
          "type": "detections",
          "attributes": {
            "confidence_threshold": 0.5,
-           "detector_name": "my_detector"
+           "detector_name": "my_vision_service"
```
Comment (stevebriskin) [inline docs/ml/vision/detection.md:250]: i believe this is right (it's the vision service), but should have <> for consistency.

**Review by stevebriskin — CHANGES_REQUESTED** [review]
defering to @bhaney

File: docs/ml/vision/detection.md (hunk 3 above)
Comment (andf-viam) [reply docs/ml/vision/detection.md:216]: The brackets in these examples represent placeholder values for the JSON Template tab, while example strings are used under the JSON Example tab. Compare the two JSON tabs [here](https://docs.viam.com/components/camera/webcam/) for example.

Comment (stevebriskin) [reply docs/ml/vision/detection.md:216]: ah, got it.

**Review by npentrel — APPROVED** [review]
Approved once Steve approves. Thanks for changing these!

File: docs/ml/vision/detection.md (hunk 4 above)
Comment (bhaney) [reply docs/ml/vision/detection.md:250]: agreed, <camera-name> has brackets below, but this one has quotes

**Review by bhaney — CHANGES_REQUESTED** [review]

File: docs/ml/vision/detection.md (hunk 4 above)
Comment (andf-viam) [reply docs/ml/vision/detection.md]: Ah I see now, updated with brackets for `my_vision_service` then, you're both right here.

**Review by bhaney — APPROVED** [review]

**Pull request merged**